src/assets/i18n/en.json5

```diff
@@ -5584,12 +5584,6 @@
 
   "admin.system-wide-alert.title": "System-wide Alerts",
 
-  "item-access-control-title": "This form allows you to perform changes to the access conditions of the item's metadata or its bitstreams.",
-
-  "collection-access-control-title": "This form allows you to perform changes to the access conditions of all the items owned by this collection. Changes may be performed to either all Item metadata or all content (bitstreams).",
-
-  "community-access-control-title": "This form allows you to perform changes to the access conditions of all the items owned by any collection under this community. Changes may be performed to either all Item metadata or all content (bitstreams).",
```

Comment:
Again, here you've accidently removed 3 more i18n keys on line 5587-5591. They must be restored